README.md

```diff
@@ -1 +1,96 @@
-# Capstone-1
\ No newline at end of file
+#  :open_book: Capstone-1
+This project creates a webpage(home and about page) for a fictional Engineer's Conference. It is based on a design by Cindy Shin in Behance.
+See a short [Project Presentation](https://www.loom.com/share/373c078d61c149fe90f650f1c9799c00)
+
+## :hammer_and_wrench: Build With
+### Tech Stack
+<details><summary>Client</summary>
+<ul>
+  <li><a href="https://html.com/">HTML</a></li>
+  <li><a href="https://www.w3.org/TR/CSS/#css">CSS</a></li>
+</ul>
+</details>
+<details><summary>Server</summary>
+</details>
+<details><summary>Database</summary>
+</details>
+
+### Key Features
+- **Mobile Version**
+  - Home page
+  - About page
+  - Hamburger menu
+
+- **Desktop Version**
+  - Home page
+  - About page
+
+<p align="right">(<a href="https://github.com/newhoteng/Capstone-1#readme">back to top</a>)</p>
+
+## :rocket: Live Demo
+- [Live link](https://newhoteng.github.io/Capstone-1/)
+
+<p align="right">(<a href="https://github.com/newhoteng/Capstone-1#readme">back to top</a>)</p>
+
+## :computer: Getting Started
+To get a local copy up and running, follow these steps.<br>
+
+**Prerequisites**<br>
+
+In order to run this project you need:
+- Text editor
+- GitHub account<br>
+
+**Setup**<br>
+
+Clone this repository to your desired folder:<br>
+```
+git clone https://github.com/newhoteng/Capstone-1.git
+```
+
+**Install**<br>
+
+Install this project with:<br>
+```
+npm install
+```
+
+
+<p align="right">(<a href="https://github.com/newhoteng/Capstone-1#readme">back to top</a>)</p>
+
+## :busts_in_silhouette: Authors
+:bust_in_silhouette: **Harriet Oteng**
+- GitHub: [@githubhandle](https://github.com/newhoteng)
+- Twitter: [@twitterhandle](https://twitter.com/HarrietOteng1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/harriet-oteng-75554666/)
+
+<p align="right">(<a href="https://github.com/newhoteng/Capstone-1#readme">back to top</a>)</p>
+
+## :telescope: Future Features
+- [x] **Desktop version**
+- [ ] **Ticketing page**
+- [ ] **Schedule page**
+
+
+<p align="right">(<a href="https://github.com/newhoteng/Capstone-1#readme">back to top</a>)</p>
+
+## :handshake: Contributing
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/newhoteng/Capstone-1/issues).
+
+<p align="right">(<a href="https://github.com/newhoteng/Capstone-1#readme">back to top</a>)</p>
+
+## :star: Show your support
+If you like this project follow me on GitHub.
+
+<p align="right">(<a href="https://github.com/newhoteng/Capstone-1#readme">back to top</a>)</p>
+
+## :pray: Acknowledgements
+I would like to thank my team mates.
+
```

Comment:
- Hey, please, let's not forget to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) to make your readme more professional.